dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -1550,6 +1612,91 @@ async fn cmd_nexus_blueprints_import(
     Ok(())
 }
 
+/// Runs `omdb nexus sagas list`
+async fn cmd_nexus_sagas_list(
+    client: &nexus_client::Client,
+) -> Result<(), anyhow::Error> {
+    // We don't want users to confuse this with a general way to list all sagas.
+    // Such a command would read database state and it would go under "omdb db".
+    eprintln!(
+        "{}",
+        textwrap::wrap(
+            "NOTE: This command only reads in-memory state from the targeted \
+            Nexus instance.  Sagas may be missing if they were run by a \
+            different Nexus instance or if they finished before this Nexus \
+            instance last started up.",
+            80
+        )
+        .join("\n")
+    );
+
+    let saga_stream = client.saga_list_stream(None, None);
+    let sagas =
+        saga_stream.try_collect::<Vec<_>>().await.context("listing sagas")?;
+
+    #[derive(Tabled)]
+    #[tabled(rename_all = "SCREAMING_SNAKE_CASE")]
+    struct SagaRow {
+        saga_id: Uuid,
+        state: &'static str,
+    }
+    let rows = sagas.into_iter().map(|saga| SagaRow {
+        saga_id: saga.id,
+        state: match saga.state {
+            SagaState::Running => "running",
+            SagaState::Succeeded => "succeeded",
+            SagaState::Failed { .. } => "failed",
+            SagaState::Stuck { .. } => "stuck",
```

Comment:
Not a blocker for the demo saga change, but I wonder if it would be nice to either include more error details here, or add a command for inspecting a saga by ID, as well? We can do that separately though

Reply:
Yeah, I think this would be good, but I think it can be done from the db state and so it probably belongs in the future db-based command.

Reply:
To add to that because it's probably not obvious: I think it's much rarer that somebody would use this subcommand instead of the db-based one, once we have it.  You'd have to be debugging something related to saga management (dispatch, recovery, the transfer that happens at expungement, etc.).  If you were debugging sagas themselves, I expect you wouldn't care where they were running.

Reply:
Yup, makes sense to me.